### PR TITLE
Apihit

### PR DESCRIPTION
**Pull request by shakerAbuDrais**
Created a new modules, that responsible for creating new game, adding players and scores to the game, calling the data from the api and displaying it on the screen.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,40 @@
...
+# Leaderboard
+
+> 
+
+
+## Built With
+
+- HTML
+- CSS
+- JavaScript
```
Comment (codepantha) [inline README.md]: - [ ] Please use this [template(https://github.com/microverseinc/readme-template) as a reference to make your README more professional. Ensure that you have all the main headers like "Getting Started".

File: src/modules/gameHandler.js [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+    headers: {
+      'Content-Type': 'application/json',
+    },
+    body: JSON.stringify({
+      name: 'MyGame',
+    }),
+  });
+  const data = await res.json();
+  localStorage.setItem('GameID', data.result.split(' ')[3]);
+}
```
Comment (codepantha) [inline src/modules/gameHandler.js]: - [ ] Please use arrow functions instead of the `function` keyword. This is a project requirement 👍

**Review by codepantha — CHANGES_REQUESTED** [review]
# STATUS: REQUIRES CHANGES ♻️ 

## Hi @shakerAbuDrais 👋, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights 🎯 
✔️ Descriptive PR
✔️ Good commit history
✔️ App working as expected
✔️ No linter errors


## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- n/a

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md [hunk 3]
```diff
@@ -1 +1,39 @@
...
+# Leaderboard
+A project to display a simple score game, using API, You push the player name and his score, and upon clicking on refresh you will get all the results that are pushed to the API. 
+
+
+## Built With
+
+- HTML
+- CSS
+- JavaScript
+
```
Comment (Ismailco) [inline README.md:10]: - [ ] As the previous reviewer suggest kindly add a Get Started section with clear instructions on how to use the app

**Review by Ismailco — CHANGES_REQUESTED** [review]

File: README.md [hunk 4]
```diff
@@ -1 +1,39 @@
...
+## Live Demo
+[Live Demo Link](https://leaderboardgame.netlify.app/)
+
```
Comment (Ismailco) [inline README.md:13]: - [x] As the previous reviewer suggest kindly add a Get Started section with clear instructions on how to use the app

Comment (Ismailco) [inline README.md:13]: - [ ] As the previous reviewer suggest kindly add a Get Started section with clear instructions on how to use the app

**Review by Ismailco — CHANGES_REQUESTED** [review]
### Hi @shakerAbuDrais, 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't been addressed yet.


 ## Highlights

✔️  No linter errors
✔️ Used Gitflow
✔️ Work documented professionally

 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md (hunk 4 above)
Comment (shakerAbuDrais) [reply README.md:13]: @Ismailco fixed can you please check, thanks a lot for your time and insights

Comment (Ismailco) [reply README.md:13]: @shakerAbuDrais, kindly request a review from the dashboard

**Review by Olamarx — APPROVED** [review]
Hi @shakerAbuDrais, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



## Highlights
- Linters Passed 👍🏼 
- Project works as expected ✔️ 
- Neat code base 😉 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**